lib/astarte_core/cql_utils.ex

```diff
@@ -167,4 +167,21 @@ defmodule Astarte.Core.CQLUtils do
 
     eid
   end
+
+  @spec realm_name_to_keyspace_name(nonempty_binary(), nonempty_binary()) :: nonempty_binary()
```

Comment:
The typespec is not correct anymore: I'd suggest this, as `""` is a valid instance id (due to backward compatibility)
```suggestion
  @spec realm_name_to_keyspace_name(nonempty_binary(), binary()) :: nonempty_binary()
```